driver-core/src/test/java/com/datastax/driver/core/ParseUtilsTest.java

```diff
@@ -60,4 +60,43 @@ public void testIsQuoted() {
         assertThat(ParseUtils.isQuoted("'''foo'''")).isTrue();
     }
 
+    @Test(groups = "unit")
```

Comment:
These tests are pretty much copy-pasted versions of the quoted tests, but figured it would be better to have a little duplication then to risk making a mistake when refactoring around common code.  Can refactor though if you like.